backtrace-library/src/main/cpp/CMakeLists.txt

```diff
@@ -55,14 +59,14 @@ find_library( # Sets the name of the path variable.
 # Specifies libraries CMake should link to your target library. You
 # can link multiple libraries, such as libraries you define in this
 # build script, prebuilt third-party libraries, or system libraries.
+list(APPEND LIBS backtrace-native)
+list(APPEND LIBS ${log-lib})
+list(APPEND LIBS crashpad_client)
+list(APPEND LIBS crashpad_util)
+list(APPEND LIBS base)
+
+if(NOT ANDROID_PLATFORM MATCHES "^android-([0-9]|1[0-9]|2[0-2])$")
```

Comment:
can we use a named feature instead of repeating the condition here?

Reply:
or unconditionally populate a list of libraries for `bun` (which could be an empty list) around L35?